recipe/bld.bat

```diff
@@ -1,5 +1,9 @@
+@echo on
 set "PYO3_PYTHON=%PYTHON%"
 
+sed -i "/lto = ""fat""/lto = ""thin""/g" Cargo.toml
+if errorlevel 1 exit 1
+
 set "CMAKE_GENERATOR=NMake Makefiles"
 set "RUSTFLAGS=-C codegen-units=4"
```

Comment:
I think using a single thread also consumes less memory. 

The output binaries compiled with a single thread are also faster.

Reply:
i.e. `codegen-units=1`?

Reply:
Yeap!